Lecture2/task1/Salad.java

```diff
@@ -0,0 +1,32 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TreeMap;
+
+public class Salad {
+    private String name;
+    private TreeMap<Integer, Vegetable> vegetables = new TreeMap<>();
```

Comment:
please read about the Maps and be sure that you always can have the unique keys
so construction won't be valid
vegetables.put(1, Vegetable);
vegetables.put(1, Vegetable);

please choose another logic for adding Vegetables

Reply:
changed Map to ArrayList